commands/utilities/health.js

```diff
@@ -0,0 +1,11 @@
+const { SlashCommandBuilder, WebSocketManager } = require("discord.js");
+
+module.exports = {
+    data: new SlashCommandBuilder()
+        .setName('health')
+        .setDescription('Checks if the bot is operational.'),
+    async execute(interaction) {
+        const sent = await interaction.reply({ content: 'Pinging...', fetchReply: true });
+        interaction.editReply(`Roundtrip latency: ${sent.createdTimestamp - interaction.createdTimestamp}ms`);
```

Comment:
Put the math in it's own line and add await
```suggestion
            const roundtripLatency = sent.createdTimestamp - interaction.createdTimestamp;
            await interaction.editReply(`Roundtrip latency: ${roundtripLatency}ms`)
```